### ISSUE TITLE
Data Attribute Facet: include All in Quick facet on filter CO

### ISSUE THREAD

**Issue by debpaul** [SpeciesFileGroup/taxonworks#3074]
### Feature or enhancement

When I want to filter on any (or some of the) **data attributes** in the project dataset, I might not know or remember what all of them are.

In filtering to create a subset of data, I would like to:
- see all the data attributes possible
- choose one or more that I want to use in this filtering step.

### Location

Task - Filter collection objects

### Screenshot, napkin sketch of interface, or conceptual description

![Picture1](https://user-images.githubusercontent.com/2990155/184949452-c6e6edbc-1406-49b3-be93-9dcadd1db202.png)

### Your role

community liaison

**Comment by proceps** [comment]
in radial annotator we have `All` tab. This should be extended to other places where attributes or tags are used.
![image](https://user-images.githubusercontent.com/5597748/184950720-0f1c6983-5228-4615-a0cf-c1d1831f4119.png)

**Comment by mjy** [comment]
Remember that you can also see all Predicates in the Manage CVTs task.

**Comment by debpaul** [comment]
> Remember that you can also see all Predicates in the Manage CVTs task.

Yes, I get that. I'm just thinking that at the time of creating a filter, I'd want to see, and then choose one or more (all in one place). In this scenario, I guess I'd have a 2nd tab open with the Manage CVTs task and then use that to help me pick Data Attributes in Filter CO

**Comment by mjy** [comment]
@debpaul you are getting at the difference b/w "browse" (I don't know what I'm looking for) and "search" (I know what I'm looking for).  Perhaps the broader question is- is the specific user interface "Filter nomenclature" the place for both (it most definitely fills the "search" role), or should we envision other ways to browse?

**Comment by jlpereira** [comment]
`All` tab is already there
